libraries/core_libs/consensus/src/vote_manager/vote_manager.cpp

```diff
@@ -699,6 +748,11 @@ std::vector<std::shared_ptr<Vote>> VoteManager::getProposeRewardVotes() {
     return {};
   }
 
+  // Add additional reward votes that were not part of 2t+1
```

Comment:
not part of 2t+1 ? All you need here is to call original method `getTwoTPlusOneVotedBlockVotes` - it does not return 2t+1 votes, it returns all founf votes for 2t+1 voted block

having to create getTwoTPlusOneVotedBlockVotesInternal because it uses the same lock is again a sign of not ideal architecture

Reply:
Done